css/style.css

```diff
@@ -0,0 +1,174 @@
+nav {
+  width: 75%;
+  min-width: 900px;
+  margin:auto;
+  color:white;
+  font-size:15px;
+  position:absolute;
+  left:50%;
+  margin-left: -37%;
+  top:110px;
+}
+nav a {
+  color: white;
+  text-decoration: none;
+}
+nav a:hover {
+  text-decoration: underline;
+}
+
+nav input{
+  max-width:105px;
+  background-color: transparent;
+  border:1px solid #2f2f2f;
+  border-radius: 15px;
+  padding-left:10px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+nav img{
+  height: 15px;
+  width: 15px;
+}
+nav ul{
+  background-color: gray;
+  background:linear-gradient(#737373, #5e5e5e 50%, #707070 50%,#8d8d8d 100%);
+  border: 1px solid black;
+  border-radius: 3px;
+  height: 30px;
+  list-style: none;
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left:0px;
+}
+nav ul li{
+  box-sizing: border-box;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  border-right:1px solid black;
+  padding-top:5px;
+}
+nav ul li:last-of-type{
+  border-right: none;
+}
+
+main{
+  padding:0;
+  margin-top:0;
+  min-width: 1024px;
+ }
+#main{
+  box-sizing: border-box;
+  background:black;
+  height: 538px;
+  background-image: url(../images/main_bg.jpg);
```

Comment:
Tip 💡: 

I noticed you used `background-image`, `background-repeat` and `background-position` properties a lot. You can shorten this into `background` property.

See [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/background) for more info.